src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -237,34 +254,25 @@ export function useSearchCurrencyLists() {
             asset => asset.address === inputAssetBridgedToSelectedChainAddress && asset.chainId === selectedOutputChainId.value
           )
         : null;
+
     const filteredBridgeAsset = bridgeAsset && filterBridgeAsset({ asset: bridgeAsset, filter: query }) ? bridgeAsset : null;
+    const isBridgeAssetUserFavorite =
+      bridgeAsset &&
+      unfilteredFavorites.some(asset => asset.mainnetAddress === bridgeAsset.mainnetAddress || asset.address === bridgeAsset.address);
```

Comment:
@christianbaroni is there a more elegant solution here? We didn't have favorite data in the `bridgeAsset` before.